workitem/simple_type.go

```diff
@@ -82,6 +82,12 @@ func (fieldType SimpleType) ConvertToModel(value interface{}) (interface{}, erro
 	case KindEnum:
 		// to be done yet | not sure what to write here as of now.
 		return value, nil
+	case KindMarkup:
+		// markup is a string
+		if valueType.Kind() != reflect.String {
+			return nil, fmt.Errorf("value %v should be %s, but is %s", value, "string", valueType.Name())
```

Comment:
I know this is new in the code but IMHO we should be using github.com/pkg/errors.Errorf() instead of fmt.Errorf() because it allows for much easier debugging later. @aslakknutsen fine by you to begin introducing this?